configure.d/1_reread_partitions.conf

```diff
@@ -23,7 +23,7 @@ apply() {
         add_function "
         static inline void cas_reread_partitions(struct block_device *bdev)
         {
-            bdev_disk_changed(bdev, false);
+            bdev_disk_changed(bdev->bd_disk, false);
```

Comment:
is this going to work on older kernels?

Reply:
It does, configure script would use "ioctl_by_bdev" implementation on 5.4 and earlier kernels.